src/webgpu/api/validation/createTexture.spec.ts

```diff
@@ -39,9 +38,19 @@ g.test('zero_size')
           'mipLevelCount',
         ] as const)
       )
-      .combine(poptions('format', ['rgba8unorm', 'rgb10a2unorm', 'bc1-rgba-unorm'] as const))
+      .combine(
+        poptions('format', [
+          'rgba8unorm',
+          'rgb10a2unorm',
+          'bc1-rgba-unorm',
+          'depth24plus-stencil8',
+        ] as const)
+      )
       .unless(
-        ({ format }) => format === 'bc1-rgba-unorm' && dimension !== '2d' && dimension !== undefined
+        ({ format }) =>
+          (format === 'bc1-rgba-unorm' || format === 'depth24plus-stencil8') &&
```

Comment:
nit: can this use the `textureDimensionAndFormatCompatible` helper as well?

Reply:
Done.